src/database.rs

```diff
@@ -599,7 +601,9 @@ impl Database {
         for t in txs.iter() {
             let tx = Tx::try_from(t.as_slice()).map_err(|_| Error::InvalidTxData)?;
 
-            let mut code = Default::default();
+            let mut code: [u8; 32] = Default::default();
+            let mut code_type: String = "wrapper".to_string();
```

Comment:
I think this is redundant with `tx_type` which already exist in the table. You want to indicate if the tx is decrypted or wrapper... etc, right ?

Reply:
I'd like to indicate what that decrypted tx actually contains

tx_transfer
tx_ibc
etc

Reply:
Ah ok. We use the `code` to indicate the tx_type but indeed you need to know it. 
So just change the default value `wrapper` with `unknown`. Wrapper is another type (decrypted, wrapper, protocol,...) and it is different from what you want (aka tx_transfer, tx_ibc, ...). Just so we don't get mixed up.